apps/carbon/src/components/Extension/Extension.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { ContentHeading } from "@cloudoperators/juno-ui-components"
+
+const Extension = ({ extension }: { extension: Extension }) => (
```

Comment:
Would it be deleted later right?

Reply:
no just the implementation will be different later.